src/llmq/options.cpp

```diff
@@ -116,28 +116,26 @@ std::map<Consensus::LLMQType, QvvecSyncMode> GetEnabledQuorumVvecSyncEntries()
 
 bool IsQuorumTypeEnabled(Consensus::LLMQType llmqType, gsl::not_null<const CBlockIndex*> pindexPrev)
 {
-    return IsQuorumTypeEnabledInternal(llmqType, pindexPrev, std::nullopt);
+    return IsQuorumTypeEnabledInternal(llmqType, pindexPrev, std::nullopt, std::nullopt);
 }
 
 bool IsQuorumTypeEnabledInternal(Consensus::LLMQType llmqType, gsl::not_null<const CBlockIndex*> pindexPrev,
-                                std::optional<bool> optDIP0024IsActive)
+                                std::optional<bool> optDIP0024IsActive, std::optional<bool> optHaveDIP0024Quorums)
 {
     const Consensus::Params& consensusParams = Params().GetConsensus();
 
     const bool fDIP0024IsActive{optDIP0024IsActive.value_or(DeploymentActiveAfter(pindexPrev, consensusParams, Consensus::DEPLOYMENT_DIP0024))};
+    const bool fHaveDIP0024Quorums{optHaveDIP0024Quorums.value_or(pindexPrev->nHeight >= consensusParams.DIP0024QuorumsHeight)};
     switch (llmqType)
     {
         case Consensus::LLMQType::LLMQ_DEVNET:
             return true;
         case Consensus::LLMQType::LLMQ_50_60:
-            if (Params().NetworkIDString() == CBaseChainParams::TESTNET) return true;
-            return !fDIP0024IsActive;
-
+            return !fDIP0024IsActive || !fHaveDIP0024Quorums ||
```

Comment:
this line should not be changed, should it?

Reply:
note `// fall through` here https://github.com/dashpay/dash/commit/b8a086d5e7d6c0307f429e3844e4331ffb036c89#diff-cf3d9716cca5e57a6033574399f5be8a184f017da2c02c414ae2771c101a2339R870

Reply:
yes, indeed, it has been changed in #5790 and reverted later in 5841 (this PR).
Both RRs are between v20.0.4 and v20.1, so, it should not affect then